standalone-metastore/src/main/sql/mysql/upgrade-2.3.0-to-3.0.0.mysql.sql

```diff
@@ -200,8 +200,18 @@ ALTER TABLE COMPLETED_TXN_COMPONENTS ADD CTC_WRITEID bigint;
 -- add a new column to support default value for DEFAULT constraint
 ALTER TABLE `KEY_CONSTRAINTS` ADD COLUMN `DEFAULT_VALUE` VARCHAR(400);
 
-ALTER TABLE `HIVE_LOCKS` CHANGE COLUMN `HL_TXNID` `HL_TXNID` bigint NOT NULL;
+ALTER TABLE `HIVE_LOCKS` MODIFY COLUMN `HL_TXNID` NOT NULL;
```

Comment:
Looks you modify the command in master code. Is it intentional or overwritten by mistake?

Reply:
reverted